Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:15.3.0-alpine3.10 as development
+WORKDIR /build
+COPY package*.json ./
+RUN npm install --only=development
+COPY . .
+RUN npm run prebuild && npm run build
+
+FROM node:15.3.0-alpine3.10 as production
+WORKDIR /app
+COPY package*.json ./
+RUN npm install --only=production
+COPY . .
```

Comment:
It's not necessary to copy sources because you need only built sources (you copy them in next step). This step could be omitted.

Reply:
Thank you, fixed :)